src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, serif;
```

Comment:
Consider using a fallback font stack for `font-family` to ensure better cross-platform compatibility. For example, `font-family: 'Roboto', 'Helvetica Neue', Arial, sans-serif;`.